### DISCUSSION TITLE
How to increase entropy source strength while running openssl command line tool?

### DISCUSSION THREAD

**Discussion by prasant-git** [openssl/openssl#24291] [Q&A] [answered]
Hi All,

As one of our application runs on **nonstop** both **itanium** and X86 hardware, 
I am trying to build openssl on these platform and link statically to our application.

I know earlier I had asked how to build openssl on nonstop platforms and come to know there is no much support related to this.
Currently though I am able to build openssl and even on x86 platform I am able to link to my application but
On Itanium platform I am getting entropy issue. 
I know hardware randomization not supported on Itanium hence I use **egd** option to configure openssl as suggested on openssl site. 
Still I get below error while running openssl command line tool.

/home/openssl/openssl-3.2.0/apps: **./openssl rand -hex 4**
00000000:error:1C8000BA:Provider routines:seed_get_seed:entropy source strength
too weak:/home/openssl/openssl-3.2.0/providers/implementations/rands/seed_src.c:
202:
00000000:error:1C8000BD:Provider routines:ossl_prov_drbg_instantiate:error retri
eving entropy:/home/openssl/openssl-3.2.0/providers/implementations/rands/drbg.c
:449:
00000000:error:1200006C:random number generator:rand_new_drbg:error instantiatin
g drbg:/home/openssl/openssl-3.2.0/crypto/rand/rand_lib.c:688:
/home/openssl/openssl-3.2.0/apps:

How to increase entropy source strength while running openssl command line tool?

**Comment by paulidale** [comment, marked as answer]
Not currently.

My attempt to implement NIST's SP 800-90x entropy strength increase was dismissed.
I don't remember the specific details.
